### ISSUE TITLE
Blur themes makes moving windows laggy

### ISSUE THREAD

**Issue by andytham** [Wox-launcher/Wox#2891]
**Describe the bug**
Selecting BlurBlack or BlurWhite, once I drop focus of Wox, moving windows of other applications is very noticeably laggy and can take up to 2 seconds to catch up to where the mouse should be.

**To Reproduce**
1. Swap to a Blur theme
2. Open Wox
3. Drop Focus
4. Drag the window of another application

**Expected behavior**
No lag

**Runtime info**
 - Windows 10 Version: 1809
 - Wox version: 1.3.881

**Comment by bao-qian** [comment]
please upload a video, so I can understand your problem.

**Comment by andytham** [comment]
https://www.youtube.com/watch?v=T4nr-lAoSCw&

I am moving the windows at the same speed and rate.

Additional info:
Only occurs when Wox is running, even if I stress my CPU.
The effect very visible when I have a lot of things running.
I assume it has something to do with the transparency effect (several of my programs utilize some sort of transparency effect too, and the lag is lessened when those are turned off).

For now I am just using a regular theme and it seems to be working just fine.

**Comment by 1139259155** [comment]
System Light  theme has the same problem

**Comment by Vicropht** [comment]
Tried to reproduce it here, didn't lag for me

**Comment by yatli** [comment]
@bao-qian take a look here: https://github.com/AvaloniaUI/Avalonia/blob/7d3c1e10aab870f22908c2e9fea3fdf22c800330/src/Windows/Avalonia.Win32/WindowImpl.cs#L357

TL;DR -- there are two ways to enable acrylic for win32 windows -- and Avalonia is using the new interface with no lags
